content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.getTimeInMillis()'
+Description: 'Returns the time in milliseconds.'
+Subjects: 
+	- 'Code Foundations'
```

Comment:
the indentations within the metadata section is incorrect. Please review one of our templates for reference.